docs/src/main/paradox/docs/releases/v1.9-release-notes.md

```diff
@@ -258,9 +261,54 @@ It is now possible to query whether the current logged in user has a specific pe
 
 @ref:[More information](../delta/api/user-permissions-api.md)
 
-## Nexus Fusion
+## Nexus Fusion 1.9 Release Notes
+
+The 1.9 release of Nexus Fusion focuses on enhancing user interaction, resource management, and overall application efficiency. This update introduces new features and key improvements across various sections of the application.
+
+### Home Page
+
+- **Global User Interface Improvements:**
+  - Multi-type filter selection in "My Data" section.
+  - Improved header menu behavior while scrolling.
+  - Enhanced uniformity in UI elements.
+
+### Data Explorer
+
+- **Enhancements:**
+  - New filtering and navigation features.
+  - Fullscreen mode for Graph Flow.
+  - Advanced mode beta for complex operations.
+  - Improved navigation with back and forward buttons.
+
+### Project Listing Page
+
+- **Resource Management and Display:**
+  - Display of resources missing paths and containing specific values.
+  - Clearer resource counts and enhanced popover information.
+  - Introduction of Data Cart logic.
+  - Improved ACL checks and permissions retrieval.
+
+### Studio Interface
+
+- **Code Editor and Studio Enhancements:**
+  - URL copying feature in Code Editor.
+  - Typo corrections and layout adjustments in studio interface.
+  - Tagging resources and splitting resource history.
+
+### Functional Improvements
+
+- **Operational Improvements:**
+  - Advanced search and filter capabilities.
+  - Bulk request handling.
+  - New resource tagging features.
+  - Enhanced error handling and feedback mechanisms.
+
+### Hotfixes and Minor Updates
 
-TODO
+- Addressing navigation speed, URL encoding in downloads, and UI glitches.
+- Improved link detection and resolution in the code editor.
+- Enhanced ID resolution for multiple resources.
```

Comment:
you can group this 3 points to "ID resolution in resource viewer/editor" and put it under "Resource View"

Reply:
In the "Resource View" category you can add also the ID resolution by pasting the encoded resource ID in URL input of the browser (you better reformulate the title 😉 )